app/src/main/java/org/fossasia/susi/ai/helper/Utils.kt

```diff
@@ -6,18 +6,22 @@ import org.fossasia.susi.ai.R
 import org.fossasia.susi.ai.rest.clients.BaseUrl
 import org.fossasia.susi.ai.rest.responses.susi.SkillData
 
-class ImageUtils {
-
+object Utils {
     private val imageLink = BaseUrl.SUSI_DEFAULT_BASE_URL + "/cms/getImage.png?"
 
     fun setSkillsImage(skillData: SkillData, imageView: ImageView) {
         Picasso.with(imageView.context)
-                .load(StringBuilder(imageLink)
-                        .append("model=" + skillData.model + "&language=" + skillData.language + "&group=" + skillData.group.replace(" ", "%20"))
-                        .append("&image=").append(skillData.image).toString())
+                .load(getLink(skillData))
                 .error(R.drawable.ic_susi)
                 .fit()
                 .centerCrop()
                 .into(imageView)
     }
-}
\ No newline at end of file
+
+    fun getLink(skillData: SkillData): String {
+        return StringBuilder(imageLink)
+                .append("model=" + skillData.model + "&language=" + skillData.language + "&group=" + skillData.group.replace(" ", "%20"))
```

Comment:
What's the use of StringBuilder if you are concatenating strings in the end?

Use kotlin string interpolation instead